tests/resources/serializers/test_ui_serializer.py

```diff
@@ -67,7 +67,6 @@ def test_ui_serializer(app, full_record_ui):
     with app.app_context():
         serialized_record = UIJSONSerializer().serialize_object_to_dict(
             full_record_ui)
-
```

Comment:
minor: not a problem, but try not to change previous things to not change much the git history.